.coveragerc

```diff
@@ -7,6 +7,8 @@ source = evalai/
 exclude_lines =
     import
     @click.*
+    pragma: no cover
```

Comment:
`pragma: no cover` has to be commented after the statement that we want to exclude from Coverage

Reply:
I'm sorry I can't really understand what you are trying to say, could you please elaborate a bit? @vkartik97

Reply:
`exclude_lines` consist of Keyword or Pattern(Regex), that is used to exclude a statement from `coverage`.
In case of `pragma: no cover`, if `coverage` comes across this keyword, the line will be excluded from the coverage calculation.
*For Example,*
```py
statement 1
statement 2 #pragma: no cover
```
Statement 2 will be ignored by `coverage.py`, thus not affecting the coverage score.
Also, you can use any other keyword like `no coverage`, `ignore: cover` etc instead of `pragma: no cover`.

Sources: 
- https://coverage.readthedocs.io/en/coverage-4.4.2/config.html#report
- https://coveralls.io/builds/19967742/source?filename=evalai/main.py, [`Line 31`](https://coveralls.io/builds/19967742/source?filename=evalai/main.py#L31) to [`Line 36`](https://coveralls.io/builds/19967742/source?filename=evalai/main.py#L36)